.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: build
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Build and install wheel
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Can you add a comment here explaining why pip is not new enough?

Reply:
Thanks for prompting this. This was a red herring when I was debugging the GH action; the real issue I was hunting was resolved in https://github.com/lsst/alert_packet/pull/15/commits/801fa738bc3991ce3046f99aea443c726babe90d - I didn't specify package_data correctly. I'll remove this upgrade line.